.github/workflows/testxpr.yml

```diff
@@ -0,0 +1,137 @@
+name: hdf5 TestExpress CI
+
+# Run TestExpress CI daily at 04:00 CDT (9:00 UTC)
+on:
+  workflow_dispatch:
+  schedule:
+    - cron: "0 9 * * *"
+
+permissions:
+  contents: read
+
+jobs:
+  build_and_test:
+    strategy:
+      matrix:
+        build_sys: ["CMake"]
```

Comment:
Will this just be Cmake for now?

Reply:
Autotools changes in pipeline once CMake works